roscpp_azure_iothub/test/testiothub.launch

```diff
@@ -0,0 +1,14 @@
+<launch>
+
```

Comment:
You can define something like:

```xml
<arg name="iot_hub_connection_string" default="abcd"/>
```

And refer it later in the `<param>`:

```xml
<param name="iot_hub_connection_string" value="$(arg iot_hub_connection_string)" />
```

And then user can replace with the string on the fly when running it. For example, `roslaunch roscpp_azure_iothub testiothub.launch iot_hub_connection_string:=<connection_string>`.

Reply:
Sure. Code updated.